### PR TITLE
Remove event file support

### LINKED ISSUE #7615: Remove event file support
### Problem

The event code supports downloading but only supports partially parsing the deprecated event file v3. It does not parse the list of items nor does it persist anything to the database. Since it's never been used and may soon be going away completely we should remove it to reduce our maintenance burden.

### Solution

Remove all event file related code

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by edwin-greene**
**Description**:

Remove support for event files

**Related issue(s)**:

Fixes #7615 

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,10 +38,6 @@ Database upgrades are now performed automatically when the mirror node component
 
 This is to ease onboarding on the integration test network, there are a few gaps in the file hash history as a result of testing which result in the parser stopping, setting this value to "X" in the `config.json` file will ensure parsing continues regardless of the hash history. This should not be set to "X" in production of course.
 
-### Addition of event parsing
```
Comment (xin-hedera) [inline CHANGELOG.md:41]: just noticed the change log. I think we should remove it as a whole, most part of is completely out of sync

**Comment by xin-hedera** [comment]
need to remove `hedera-mirror-importer/src/test/resources/data/eventsStreams`

File: CHANGELOG.md (hunk 1 above)
Comment (steven-sheehy) [reply CHANGELOG.md:41]: I'm fine with removing the changelog

Comment (edwin-greene) [reply CHANGELOG.md:41]: Thanks I have removed it

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Pull request merged**